src/foam/u2/filter/properties/StringFilterView.js

```diff
@@ -220,7 +219,7 @@ foam.CLASS({
             onKey: true
           })
           .end()
-          .start('p')
+          .start('h4')
```

Comment:
We don't use these anymore. Could you add the class `h500/h600` instead

Reply:
Added h600, looks  like this now
<img width="1680" alt="Screen Shot 2021-06-10 at 12 36 47 PM" src="https://user-images.githubusercontent.com/26717171/121563610-94254c00-c9e8-11eb-8921-4e9746b8013e.png">